assets/wizards/setup/index.js

```diff
@@ -0,0 +1,275 @@
+		const { setError } = this.props;
+		const params = {
+			path: `/newspack/v1/wizard/newspack-setup-wizard/complete`,
+			method: 'post',
+		};
+		return new Promise( ( resolve, reject ) => {
+			return apiFetch( params )
+				.then( response => {
+					this.setState( { setupComplete: true }, () => resolve( response ) );
+				} )
+				.catch( error => {
+					console.log( '[Complete Setup Error]', error );
+					setError( error );
+					reject( error );
+				} );
+		} );
+	};
+
+	isConfigurationComplete = () => {
+		const { pluginInfo } = this.state;
+		return (
+			pluginInfo[ 'jetpack' ] &&
+			pluginInfo[ 'jetpack' ].Configured &&
+			pluginInfo[ 'google-site-kit' ] &&
+			pluginInfo[ 'google-site-kit' ].Configured
+		);
+	};
+
+	getFirstPluginToConfigure = () => {
+		const { pluginInfo } = this.state;
+		if ( ! pluginInfo[ 'jetpack' ] || ! pluginInfo[ 'google-site-kit' ] ) {
+			return null;
+		}
+		if ( ! pluginInfo[ 'jetpack' ].Configured ) {
+			return { handoff: 'jetpack' };
+		}
+		if ( ! pluginInfo[ 'google-site-kit' ].Configured ) {
+			return { handoff: 'google-site-kit' };
+		}
+		return null;
+	};
+
+	/**
+	 * Render.
+	 */
+	render() {
+		const {
+			installedPlugins,
+			installationComplete,
+			pluginInfo,
+			profile,
+			countries,
+			currencies,
+			setupComplete,
+		} = this.state;
+		const installProgress = Object.keys( installedPlugins ).length;
+		const installTotal = REQUIRED_PLUGINS.length;
+		return (
+			<Fragment>
+				<NewspackLogo width="240" className="newspack-logo" />
+				<HashRouter hashType="slash">
+					<Switch>
+						<Route
+							path="/"
+							exact
+							render={ routeProps => (
+								<Welcome
+									buttonText={ __( 'Get started' ) }
+									buttonAction={ {
+										href: '#/about',
+										onClick: () => this.updateProfile(),
+									} }
+									profile={ profile }
+								/>
+							) }
+						/>
+						<Route
+							path="/about"
+							render={ routeProps => (
+								<About
+									headerText={ __( 'About your publication' ) }
+									subHeaderText={ __(
+										'Share a few details so we can start setting up your profile'
+									) }
+									buttonText={ __( 'Continue' ) }
+									buttonAction={ {
+										href: '#/newsroom',
+										onClick: () => this.updateProfile(),
+									} }
+									profile={ profile }
+									currencies={ currencies }
+									countries={ countries }
+									updateProfile={ ( key, value ) => {
+										this.setState( { profile: { ...profile, [ key ]: value } } );
+									} }
+								/>
+							) }
+						/>
+						<Route
+							path="/newsroom"
+							render={ routeProps => (
+								<Newsroom
+									headerText={ __( 'Tell us about your Newsroom' ) }
+									subHeaderText={ __(
+										'The description helps set the stage for the step content below'
+									) }
+									buttonText={ __( 'Continue' ) }
+									buttonAction="#/configure-plugins"
+									buttonDisabled={ ! installationComplete }
```

Comment:
I agree, we'll need to communicate to the user what is happening and why they can't progress. (Hopefully the plugin installation process is completed before the user is done filling out the forms in most cases.)

We could look at a few options here. Let's start with testing your suggestion: using explanatory text in the "inactive" button. Another option could be adding a tooltip, though I think that's not a very common pattern on an action button. For A11y, we might need to add screenreader text since inactive buttons are often skipped by screen readers.

Reply:
I've done a version of this in https://github.com/Automattic/newspack-plugin/pull/114/commits/4e0ac93145eb2b47f0e67ef050532be7d7c03067